pr/payjs-sandbox/pr.js

```diff
@@ -45,7 +45,7 @@ async function payButtonClickHandler() {
     // Please use your own Google Pay merchant ID.
     paymentDataRequest.merchantInfo = {
       merchantName: 'Rouslan Solomakhin',
-      merchantId: '00184145120947117657',
+      merchantId: 'BCR2DN6TXDBYXAJ7',
```

Comment:
Is it possible to add a comment about the source of this merchant ID string?

. Also, please update the merchant ID in `/pr/gp2-sandbox/pay.js`.

Reply:
(Or you can let me know the source of the merchant ID either in here or in chat, and I can submit this patch and then make any more updates myself. -- Let me know what works best for you.)